torchvision/models/detection/rpn.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         self.sizes = sizes
         self.aspect_ratios = aspect_ratios
         self.cell_anchors = None
-        self._cache = {}
+        # self._cache = {}
```

Comment:
Even if we uncomment all the caching behavior, starting with the 3rd run (cache miss, cache hit, *cache hit*) we hit the compilation cache right? So performance would be fine?

Reply:
Yup exactly, I really just disabled this for now because it makes prototyping a little bit faster for me at the moment, as described in the commit message :)